test/functional/Valhalla/playlist.xml

```diff
@@ -40,4 +40,23 @@
 			<subset>11</subset>
 		</subsets>
 	</test>
+	<test>
+		<testCaseName>ValueTypeTests</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+		--add-opens java.base/jdk.internal.misc=ALL-UNNAMED \
+	-cp $(Q)$(LIB_DIR)$(D)asm-all.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)ValhallaTests.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames ValueTypeTests \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>Valhalla</subset>
```

Comment:
Yes, at least for now

Reply:
Is the feature not ready in regular JDK? I thought Valhalla is merged into the regular JDK. Ideally, we do not want to add any test that our automation builds will never execute.

Reply:
The nestmates part of Valhalla as been merged. Valhalla is large ongoing project that will span the next 2 - 3 years.  
  
We need the tests checked in as it will make it easier to collaborate with the JIT team.

Ideally, we would be running these tests as part of build automation, however, these tests require a custom build phase and that work has not been done yet. The in the mean time we will be running it manually.

Reply:
ok, I got it.
Another related question, is it intended to compile this test in JDK11+? If not, we need to add check in https://github.com/eclipse/openj9/blob/1b946657cdba269edc7a95e518a43bfcb20a70e4/test/functional/Valhalla/build.xml#L80

Reply:
No it is not, only if `JDK_VERSION=Valhalla` is set.

I think we still need that line as it is because the nestmates tests are still part of Valhalla. Still need to move them out, https://github.com/eclipse/openj9/issues/2822

Reply:
yep, I mean we need to add an additional check in build.xml so that we compile ValueTypeTests only if `JDK_VERSION=Valhalla` is set.

Reply:
I have modified this. This PR now depends on https://github.com/eclipse/openj9/pull/3501